InnerEye/ML/run_ml.py

```diff
@@ -571,8 +571,12 @@ def register_model(self,
                 run_to_register_on = RUN_CONTEXT
                 logging.info(f"Registering the model on the current run {run_to_register_on.id}")
             logging.info(f"Uploading files in {final_model_folder} with prefix '{artifacts_path}'")
-            final_model_folder_relative = final_model_folder.relative_to(Path.cwd())
-            run_to_register_on.upload_folder(name=artifacts_path, path=str(final_model_folder_relative))
+
+            if not is_amulet_job():
```

Comment:
I find it easier to read if the condition is not inverted - you anyway cover both cases?

Reply:
Agreed + updated!